Makefile

```
@@ -205,6 +205,10 @@ install-automation-requirements: ## Install automation-requirements relevant for
 		$(MLRUN_PIP_NO_CACHE_FLAG) \
 		-r automation/requirements.txt
 
+.PHONY: patch-remote
+patch-remote: ## Deploy local mlrun-api code on a remote system (see automation/patch_igz/README.md)
```

It's not only the API
```suggestion
patch-remote: ## Deploy local code on a remote system (see automation/patch_igz/README.md)
```

---

Fixed - [CR comments](https://github.com/mlrun/mlrun/pull/9044/commits/85c5c7c3779c60b8b85915c302918bbb20f55e93)